src/components/NFTPortfolio/UserInfo.vue

```diff
@@ -5,19 +5,49 @@
       {{ likerId | ellipsis }}
     </Label>
     <div v-if="likerDescription" :class="['w-full', 'h-[1px]', 'bg-shade-gray', 'my-[16px]']" />
-    <Label preset="p6" class="break-all font-200">
+    <Label preset="p6" class="break-all font-200 my-[28px]">
       {{ likerDescription }}
     </Label>
-    <div :class="['w-full', 'h-[1px]', 'bg-shade-gray', 'my-[16px]']" />
-    <NFTPortfolioState 
+    <div class="flex justify-between w-[44px] mx-auto mb-[16px] text-shade-gray">
+      <IconEllipse />
+      <IconEllipse />
+      <IconEllipse />
+    </div>
+    <NFTPortfolioState
       :collected-items="collectedItems"
       :created-class-ids="createdClassIds"
     >
       <template v-slot="stats">
-        <Label preset="p6" class="font-200">Collections: {{ stats.collectedCount }}</Label>
-        <Label preset="p6" class="font-200">Total Value: {{ stats.collectedAmount }}</Label>
-        <Label preset="p6" class="font-200">Creations: {{ stats.createdCount }}</Label>
-        <Label preset="p6" class="font-200">Minted: {{ stats.createdCollectedCount }}</Label>
+        <div class="grid w-full grid-cols-2 cursor-default gap-x-8 gap-y-4 text-medium-gray">
```

Comment:
Can add class directly to the <NFTPortfolioState> without wrapping an extra `<div>` here